plugins/aws/documentDB/docdbClusterBackupRetention.js

```diff
@@ -0,0 +1,64 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'DocumentDB Cluster Backup Retention',
+    category: 'DocumentDB',
+    domain: 'Databases',
+    description: 'Ensure that your Amazon DocumentDB clusters have set a minimum backup retention period.',
+    more_info: `DocumentDB cluster provides feature to retain incremental backups between 1 and 35 allowing
+                you to quickly restore to any point within the backup retention period. Ensure that you have
+                sufficient backup retention period configured in order to restore your data in the event of failure.`,
+    recommended_action: 'Modify DocumentDb cluster to configure sufficient backup retention period.',
+    link: 'https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-modify.html',
+    apis: ['DocDB:describeDBClusters'],
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var RECOMMENDED_THRESHOLD = 7;
```

Comment:
We should add a setting for this. Default can be 7. Regex should only allow 1-35